pandas/_testing.py

```diff
@@ -2758,3 +2759,28 @@ def get_cython_table_params(ndframe, func_names_and_expected):
             if name == func_name
         ]
     return results
+
+
+def get_op_from_name(op_name: str) -> Callable:
```

Comment:
IIRC we have something similiar in internals, maybe we should consolidate (just a thought) and certainly. in a followon